src/ReadingBuddy.java

```diff
@@ -1227,16 +1368,22 @@ public static void viewBookGroup(BookGroup currentGroup){
             String bookBuddyName = currentGroup.getBookBuddyName(currentUser.getUsername());
             System.out.println("\n\n\nBook: " + currentGroup.getTitle() + " by " + currentGroup.getAuthor());
             System.out.println("Book Buddy: " + bookBuddyName);
-            System.out.println("Your current page: " + currentGroup.getUserPageNum(currentUser.getUsername()));
-            System.out.println(bookBuddyName + "'s current page: " + currentGroup.getUserPageNum(bookBuddyName));
+            //NEW SPRINT3: ternary operator to display page number or "Done" depending on if user done or not (left old commented out until tested)
+            System.out.println("Your current page: " + (currentGroup.getUserIsDone(currentUser.getUsername())? "Done" : currentGroup.getUserPageNum(currentUser.getUsername())));
+            System.out.println(bookBuddyName + "'s current page: " + (currentGroup.getUserIsDone(bookBuddyName) ? "Done" : currentGroup.getUserPageNum(bookBuddyName)));
+            //System.out.println("Your current page: " + currentGroup.getUserPageNum(currentUser.getUsername()));
```

Comment:
ignore the couple commented out lines, that was the stuff and i forgot to delete it